common/api/gas.ts

```diff
@@ -1,4 +1,5 @@
 import { checkHttpStatus, parseJSON } from './utils';
+import { Omit } from 'react-redux';
```

Comment:
Does this need to be imported? I see it's not here https://github.com/MyCryptoHQ/MyCrypto/pull/1263/files#diff-0fb63f2ecec1a1d8998af37ee70ac8e7R20

Reply:
It is imported in that file too